### PR TITLE
Upgrade to FontAwesome 4.7.0

### LINKED ISSUE #217: Upgrade fontawesome

### PR DESCRIPTION
**Pull request by iranzo**
Closes #217

### CODE REVIEW

**Comment by iranzo** [comment]
@talha131 this was merged already right? Should we close it?

**Comment by talha131** [comment]
No. It hasn't been merged. I was hoping to upgrade to the latest fontawesome. But that shouldn't block this PR.

I have requested a change. Please do it then I will merge the PR.

**Comment by iranzo** [comment]
Done @talha131

**Comment by talha131** [comment]
@iranzo one change I requested is still open.

**Comment by iranzo** [comment]
@talha131 what is missing? PR has both the changelog and the updated file

File: CHANGELOG.md [hunk 1]
```diff
@@ -9,6 +9,7 @@ Version 2.2
 * Changed: Reading Time is displayed only if it is greater than `READING_TIME_LOWER_LIMIT` variable (default 4 min)
 * Changed: Project documentation has been moved inside the Elegant repository
 * Removed: Google Plus author
+* Update FontAwesome to version 4.7.0
```
Comment (talha131) [inline CHANGELOG.md]: @iranzo this should come under "To Be Released" heading. And please change it to

```
New: FontAwesome updated to version 4.7.0
```

**Comment by talha131** [comment]
> @talha131 what is missing?

My bad. After requesting change, I missed to click on "Finish review" so it was visible to me but not to you. The requested change should be visible now.

**Comment by iranzo** [comment]
Done!

**Pull request merged**